Assets/Scripts/Enemy/EnemyBehaviour.cs

```diff
@@ -54,32 +54,32 @@ IEnumerator Cooldown()
         NavAgentMove = info.navAgentMove;
         GetAimDirection = info.aim;
         info.navAgentSetup(new Vector3(0, 0, 0));
-        
+
         return true;
     }
-    
+
     // Start is called before the first frame update
-    void Start() {}
+    void Start() { }
 
     // Update is called once per frame
     void Update()
     {
         if (!_initialized) return;
 
-        Animation anim = gameObject.transform.GetChild(0).GetComponent<Animation>();
+        Animation anim = gameObject.transform.childCount > 0 ? gameObject.transform.GetChild(0).GetComponent<Animation>() : null;
         // Look. Determine look direction.
-        _direction = (_targetTransform.position - transform.position).normalized;
-        _direction.y = 0;
-        _lookRotation = Quaternion.LookRotation(_direction);
-        transform.rotation = _lookRotation;
-        
+        // _direction = (_targetTransform.position - transform.position).normalized;
+        // _direction.y = 0;
+        // _lookRotation = Quaternion.LookRotation(_direction);
+        // transform.rotation = _lookRotation;

```

Comment:
was this the part that instantly rotated our enemies depending on the player position?

Reply:
Yup.